bgpd/bgp_vty.c

```diff
@@ -6413,8 +6413,8 @@ DEFUN (clear_bgp_ipv6_safi_prefix,
        "Clear bestpath and re-advertise\n"
        "IPv6 prefix\n")
 {
-	int idx_safi = 3;
-	int idx_ipv6_prefixlen = 5;
+	int idx_safi = 4;
```

Comment:
This is the wrong approach.  safi can be 3 or 4.  I would recommend searching for the safi string.

Reply:
done